docs/content/guides/developer/sui-101/working-with-ptbs.mdx

```diff
@@ -1,6 +1,25 @@
 ---
 title: Working with Programmable Transaction Blocks
-draft: true
 ---
 
-Content coming soon
\ No newline at end of file
+Programmable transaction blocks (PTBs) are key elements of the Sui ecosystem. Understanding PTBs and using them correctly are key fundamentals to creating efficient and cost-effective smart contracts. See [Programmable Transaction Blocks](../../../concepts/transactions/prog-txn-blocks.mdx) to learn about the structure of PTBs on Sui.
+
+The topics in this section focus on effectively utilizing PTBs in your smart contracts.
+
+## Simulating References
+
+The [`borrow` module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/sui-framework/borrow.md) of the Sui framework offers some features you can use when your PTBs use objects by reference. 
+
+Go to [Simulating References](./simulating-refs.mdx).
+
+## Coin Management
```

Comment:
Coin management is less about performance and more to do with the common patterns people use for accepting coins in smart contracts, and how to interface with them from a PTB.  E.g. instead of accepting a mutable reference to a coin, functions typically accept a coin by value, and expect it to contain the right balance (which they check), and on the PTB side, we use SplitCoins and MergeCoins to create the coin with the right balance in the same transaction (and also can use the GasCoin directly if we are dealing with SUI).